src/test/java/uk/gov/hmcts/reform/unspec/sampledata/CaseDataBuilder.java

```diff
@@ -429,6 +434,14 @@ public CaseDataBuilder atStateClaimDraft() {
             .build();
         claimType = ClaimType.PERSONAL_INJURY;
         personalInjuryType = ROAD_ACCIDENT;
+        applicantSolicitor1PbaAccounts = DynamicList.builder()
+            .value(DynamicListElement.builder().label("PBA0077597").build())
+            .build();;
```

Comment:
Extra semicolon here
```suggestion
            .build();
```